packages/node_modules/@webex/plugin-meetings/src/reachability/index.js

```diff
@@ -174,7 +179,7 @@ export default class Reachability {
       peerConnection.begin = Date.now();
       peerConnection.setLocalDescription(description);
 
-      return this.iceGatheringState(peerConnection)
+      return this.iceGatheringState(peerConnection, cluster.isVideoMesh ? VIDEO_MESH_TIMEOUT : DEFAULT_TIMEOUT)
```

Comment:
I can't see a test for this

Reply:
yeah the reachability code in SDK has no tests at all, I can add it